.github/CODEOWNERS

```diff
@@ -0,0 +1,237 @@
+/.github/ @elastisys/goto-pipeline/qa/release
+
+/bin/ @elastisys/goto-scripts
+
+/changelog/ @elastisys/goto-pipeline/qa/release
+
+/completion/ @elastisys/goto-scripts
+
+
+#
+# helmfile.d/charts/
+#
+/helmfile.d/charts/autoscaling-monitoring/ @elastisys/goto-kubernetes
+/helmfile.d/charts/calico-accountant/ @elastisys/goto-kubernetes
+/helmfile.d/charts/calico-default-deny/ @elastisys/goto-kubernetes
+/helmfile.d/charts/calico-felix-metrics/ @elastisys/goto-kubernetes
+/helmfile.d/charts/cluster-admin-rbac/ @elastisys/goto-security
+/helmfile.d/charts/external-dns-endpoints/ @elastisys/goto-ingress/auth
+/helmfile.d/charts/external-dns-secrets/ @elastisys/goto-ingress/auth
+/helmfile.d/charts/falco-psp-rbac/ @elastisys/goto-security
```

Comment:
Is this chart even used nowadays? :thinking:

Reply:
No it seems to just reference the old PSPs. So it can be removed. But I think it makes more sense to remove it in a separate PR. Will fix that later.